### PR TITLE
add new company - WeMakeMVP

### PR DESCRIPTION
**Pull request by genndy007**
📝Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]`

ℹ️Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

- [ ] This PR contains housekeeping only (URL edits, copy changes etc) - if check, delete other lines that don't apply.
- [x] I am an employee of the company mentioned and confirm all included details are correct.
- [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/main/company-profiles/example.md) is included - This is __required__ for new additions.
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/main/CONTRIBUTING.md).

This Pull Request should adhere to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/main/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -628,6 +628,7 @@ Name | Website | Region
 [Webikon](/company-profiles/webikon.md) | https://www.webikon.sk/en/ | Worldwide
 [Wells Fargo](/company-profiles/wells-fargo.md) | https://www.wellsfargo.com/ |
 [wemake.services](/company-profiles/wemake-services.md) | https://wemake.services/ | Worldwide
+[WeMakeMVP](/company-profiles/wemakemvp.md) | https://www.wemakemvp.com/ | Ukraine & Worldwide
```
Comment (dougaitken) [inline README.md:631]: Ukraine or Worldwide?

File: company-profiles/wemakemvp.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+The core team consists of 5 specialists.
+
+## Remote status
+
+Fully-remote
+
+## Region
+
+Ukraine (Worldwide)
```
Comment (dougaitken) [inline company-profiles/wemakemvp.md:23]: Can you clarify this part, please?

**Review by dougaitken — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (genndy007) [reply README.md]: Changed to Worldwide, our company is ready to accept people from all around the world. The thing is that founders are based in Ukraine.

File: company-profiles/wemakemvp.md (hunk 2 above)
Comment (genndy007) [reply company-profiles/wemakemvp.md]: Clarified this, added that founder is in Ukraine but the company accepts coworkers from all the globe.

**Comment by genndy007** [comment]
Fixed parts you asked, please review PR again :)

**Review by dougaitken — APPROVED** [review]

**Comment by dougaitken** [comment]
Thanks for contributing @genndy007, it is appreciated

**Pull request merged**